src/ORM/Association/BelongsTo.php

```diff
@@ -33,6 +33,13 @@ class BelongsTo extends Association
     use SelectableAssociationTrait;
 
     /**
+     * Valid strategies for this type of association
+     *
+     * @var array
+     */
+    protected $_validStrategies = [parent::STRATEGY_JOIN];
```

Comment:
Thanks, I think that should be reflected in the docs then. Right now there's no `strategy` info in the `BelongsTo` or `HasOne` sections.